### DISCUSSION TITLE
2.0.0版本按用户手册配置processor_filter_regex_native，启动报错

### DISCUSSION THREAD

**Discussion by EEElevation** [alibaba/ilogtail#1392] [Help] [answered]
采集配置：
![image](https://github.com/alibaba/ilogtail/assets/52227492/ded2e815-a112-473b-8d22-d89be225916c)
日志报错信息：
![image](https://github.com/alibaba/ilogtail/assets/52227492/a446cc61-892b-4657-8bfd-ecfa72738c08)
用户手册：https://ilogtail.gitbook.io/ilogtail-docs/plugins/processor/native/processor-filter-regex-native

**Comment by EEElevation** [comment]
实测配置修改为这样 启动正常
![image](https://github.com/alibaba/ilogtail/assets/52227492/981c64b4-505b-42e1-9d8e-f54b56c894bf)
yonghushouc
用户手册需要更新

**Comment by quzard** [comment, marked as answer]
这个确实文档的疏忽。最新的代码里改动了这个参数（也就是文档里的FilterKey和FilterRegex），但没有发版本，目前发的版本还是Include参数
